scripts/testacc_setup.sh

```diff
@@ -17,7 +17,7 @@ openssl req \
 # Create auth
 mkdir -p "$(pwd)"/scripts/testing/auth
 # Start registry
-docker run --rm --entrypoint htpasswd registry:2 -Bbn testuser testpwd > "$(pwd)"/scripts/testing/auth/htpasswd
+docker run --rm --entrypoint htpasswd registry:2.6.2 -Bbn testuser testpwd > "$(pwd)"/scripts/testing/auth/htpasswd
```

Comment:
I assume the removal of the `htpasswd` binary made you pin the version? (https://github.com/docker/docker.github.io/issues/11060)

Reply:
Yes, looked like the least intrusive way to fix the testing.
Another alternative would be to alter test bash scripts to add apache2-utils before generating credentials.